nextcord/ext/commands/core.py

```diff
@@ -1177,14 +1177,18 @@ async def can_run(self, ctx: Context) -> bool:
                 if local_check is not None:
                     ret = await nextcord.utils.maybe_coroutine(local_check, ctx)
                     if not ret:
+                        ctx.command = original
```

Comment:
Doesn't this happen already do to the finally block?

Reply:
yes it does, `copy.copy` now actually does the work of getting around the race condition